packages/server-wallet/package.json

```diff
@@ -57,6 +57,7 @@
     "eslint-plugin-jest": "23.6.0",
     "eslint-plugin-prettier": "3.3.1",
     "eslint-plugin-unicorn": "21.0.0",
+    "exponential-backoff": "^3.1.0",
```

Comment:
Did you consider `'backoff'`? It's already a (dev) dependency.

Reply:
I did briefly look at backoff but I didn't realize it was something we were already using. Taking a look now I can't seem to find it as a dependency anywhere?

Reply:
It's a dependency of `ganache-core`, so it's already in `yarn.lock`.

I looked briefly at it, and maybe it doesn't fit our use case -- there's an outstanding issue requesting that the backoff function be called without delay, and I think we want that feature, for instance.